### PR TITLE
GitHub: create issue templates

### PR DESCRIPTION
**Pull request by EmileRolley**
Add issue templates for bug reports, feature requests and questions.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/question.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+## Current configuration
+
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+## Additional context
+
+Add any other context or screenshots about the inquiery.
```
Comment (yurug) [inline .github/ISSUE_TEMPLATE/question.md:30]:
```suggestion
Add any other context or screenshots about the inquiry.
```

File: .github/ISSUE_TEMPLATE/question.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+> Related issue(s) or PR(s): 
+> Related project scope(s) (ex: client, CSS, grading, etc...) : 
+> Related user(s): 
+
+## The question
+
+A clear and concise description of what you want to know.
+
+###  What has already been tested
```
Comment (yurug) [inline .github/ISSUE_TEMPLATE/question.md:18]:
```suggestion
### What has already been tested
```

**Review by yurug — COMMENTED** [review]
These templates seem fine, at least for the first version. The usage will show if we need refinements.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+3. Scroll down to '....'
+4. See error
+
+### Expected behavior
+
+A clear and concise description of what you expected to happen.
+
+### Screenshots
+If applicable, add screenshots to help explain your problem.
+
```
Comment (maiste) [inline .github/ISSUE_TEMPLATE/bug_report.md:33]:
```suggestion
### Screenshots

If applicable, add screenshots to document your problem.
```

**Comment by maiste** [comment]
@EmileRolley, you might need to rewrite your git history before @yurug merge. Maybe only one single commit is needed.

**Comment by EmileRolley** [comment]
> @EmileRolley, you might need to rewrite your git history before @yurug merge. Maybe only one single commit is needed.

I think using the `Squash and merge` option is good enough no?

**Pull request merged**